huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go

```diff
@@ -606,6 +636,17 @@ func resourceInstanceUpdate(ctx context.Context, d *schema.ResourceData, meta in
 		}
 	}
 
+	if d.HasChange("action") {
+		action := d.Get("action").(string)
+		instanceId := d.Get("instance_id").(string)
```

Comment:
if instanceId == "" {
  return diag.Errorf("editing `action` is currently not supported because of a failure in getting `instance_id`")
}